### ISSUE TITLE
Adding an option to disappear while hovering

### ISSUE THREAD

**Issue by Snapyou2** [BlafKing/spicetify-cat-jam-synced#15]
Pretty explicit, the cat can be a bit annoying while looking in the library and is to small in the bottom bar. An option to make it automatically more transparent while hovering the mouse over it would be awesome !

Thanks for this extension by the way, it makes me vibing ehe

**Comment by Nuzair46** [comment]
good suggestion. i will look into this

**Comment by prediscover** [comment]
Yes! I agree with OP. This is sorely needed since the bottom of the library hidden & not accessible while the Cat is Jamming.
